kallisto/urls.py

```diff
@@ -7,6 +7,7 @@
 urlpatterns = patterns(
     '',
     url(r'^$', 'apps.homepage.views.homepage', name='homepage'),
+    url(r'^help$', 'apps.homepage.views.help', name='help'),
```

Comment:
I didn't know you could do that. I was looking for this the other day for a different project, but I just kept finding old references to `direct_to_template`.